app/ui-react/syndesis/src/modules/data/pages/VirtualizationSqlClientPage.tsx

```diff
@@ -1,10 +1,13 @@
 import { RestDataService } from '@syndesis/models';
 import { WithRouteData } from '@syndesis/utils';
+// tslint:disable-next-line:no-implicit-dependencies
```

Comment:
dont think you need this tsLint line?...

Reply:
see below